.eslintrc.json

```diff
@@ -1,61 +1,6 @@
 {
```

Comment:
Some of this I will probably put back.

Reply:
I'm indifferent tbh. Only reason I "changed" it is because I nuked the original dir and used the `next lint` defaults 😆 

Generally I like to "use the defaults of someone who cares more" for things like this, Prettier, TSConfig, etc, but if you have stronger preferences go nuts